e2e/helpers.go

```diff
@@ -0,0 +1,57 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/logging"
+
+	"github.com/castai/terraform-provider-castai/castai/sdk"
+)
+
+func createClient(apiURL, apiToken string) (*sdk.ClientWithResponses, error) {
```

Comment:
this is a bit duplicate with the castai config 🤔 maybe we could re-use the client creation part?